parameter-object/src/test/java/com/iluwatar/parameter/object/ParameterObjectTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * The MIT License
+ * Copyright © 2014-2019 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.iluwatar.parameter.object;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ParameterObjectTest {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ParameterObjectTest.class);
+
+  @Test
+  public void testForDefaultSortBy() {
+    //Creating parameter object with default value for SortBy set
+    ParameterObject params = ParameterObject.newBuilder()
+        .withType("sneakers")
+        .sortOrder(SortOrder.DESC)
+        .build();
+
+    assertEquals(ParameterObject.DEFAULT_SORT_BY, params.getSortBy(),
+        "Default SortBy is not set.");
+    LOGGER.info("SortBy Default parameter value is set during object creation as no value is "
+        + "passed.");
```

Comment:
Do not use concatenation in logger.

_For Syntax use this:_
```suggestion
    LOGGER.info("SortBy Default parameter value is set during object creation as no value is ",
        "passed.");
```

Reply:
Do you mean to use this method - info(String format, Object arg1, Object arg2)?

Reply:
yes